### ISSUE TITLE
need master doc

### ISSUE THREAD

**Issue by soapstain22** [soapstain22/war#29]
basic suggestions
- [ ] how does difficulty operate
- [ ] how does the save system work
- [ ] how does the gameplay loop work
- [ ] how does the main gui look
- [ ] what are the controls
- [ ] will there be more than 1 playable character
- [ ] should we develop for it to become a multiplayer game at some point 
come up with more

**Comment by FPumpkin** [comment]
1. difficulity would be selectable - since there are 3 types of guys (innocent, easily raged, law) the number of people that can spawn will be increased and the weapon selection pool will increase (both would increase at same time) 
probably: harmless > average > frighting > fun only begins > (unlockable after finishing that one <) guns are for babies (only fists and chainsaw, everyone is enraged)
2. autosave after every location load, you can save anytime when music aint playing (not in fight)
3.  open world, you can go where you want (if location isnt closed) but some features unlock only after specific days passed
4. we have to come up with custom font (prob ask bozo for everything ngl hes cool with ui design)
5. check miro
6. no
7. for like coop? maybe but not now
